src/manager.ts

```diff
@@ -112,6 +112,89 @@ export class Manager implements vscode.TreeDataProvider<string | FileSystemConfi
         }
         root = root.replace(/^~/, home.replace(/\/$/, ''));
       }
+      
+      vscode.debug.registerDebugConfigurationProvider("*", {
```

Comment:
I'm not sure we're supposed to use these register methods dynamically, especially without ever unsubscribing them